modules/sdk/src/test/resources/ratings.sample.csv

```diff
@@ -0,0 +1,4 @@
+698c4c20b6975ff2bf61b82b078d0253, -1
+aadeb50882008f5796fa0c487a8c17da, 0
```

Comment:
These are not valid peer ids

Reply:
Replaced with ids from `peerGen`.

Reply:
which are also not valid peer ids

Reply:
But they were taken from `peerGen` the arbitrary for `Peer`s. If those are not correct, then is that generator valid? Where should I be getting valid peer ids from?